app/components/computed-props.js

```diff
@@ -50,11 +51,88 @@ export default Component.extend({
    * Please replace these props below with computed props
    * @type {Array}
    */
-  userNames: [],
-  bmwusers: [],
-  whiteCarsUsers: [],
-  usersWith1Car: [],
-  userswithMoreThanOneCar: [],
-  allMakes: [] // no repeat of makes
+  userNames: computed('users', function () {
```

Comment:
that's a `computed.map` computed property https://api.emberjs.com/ember/2.15/namespaces/Ember.computed